contracts/test/integration/IntegrationTestPCVGuardian.sol

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {PCVGuardian} from "../../pcv/PCVGuardian.sol";
+import {TribeRoles} from "../../core/TribeRoles.sol";
+import {ICore} from "../../core/ICore.sol";
+import {IVolt} from "../../volt/Volt.sol";
+import {IPCVDeposit} from "../../pcv/IPCVDeposit.sol";
+
+import {DSTest} from "../unit/utils/DSTest.sol";
+import {Vm} from "../unit/utils/Vm.sol";
+
+contract IntegrationTestPCVGuardian is DSTest {
+    PCVGuardian private pcvGuardian;
+
+    ICore private core = ICore(0xEC7AD284f7Ad256b64c6E69b84Eb0F48f42e8196);
+    IVolt private fei = IVolt(0x956F47F50A910163D8BF957Cf5846D573E7f87CA);
+
+    IPCVDeposit private pcvDeposit =
+        IPCVDeposit(0x4188fbD7aDC72853E3275F1c3503E170994888D7);
+
+    Vm public constant vm = Vm(HEVM_ADDRESS);
+
+    address public voltDeployer = 0x25dCffa22EEDbF0A69F6277e24C459108c186ecB;
```

Comment:
we should really move this address out into the fixtures file so we don't have to copy paste it around everywhere